src/index.js

```diff
@@ -80,8 +80,7 @@ const shouldDisplay = () => {
 						resolve(result);
 					} else {
 						const timeout = setTimeout(() => {
-							const result = handleConsentResult(vendorList, undefined);
-							resolve(result);
+							resolve({ display: false });
```

Comment:
Shouldn't we also change it in lines 104-106?

Reply:
Good point, I think we should.